spec/std/isa/ext/V.yaml

```
@@ -50,3 +50,33 @@ params:
 
       # if HW is writing VS, then Dirty (3) better be a supported value
       assert MSTATUS_VS_LEGAL_VALUES.include?(3) if ext?(:V) && (HW_MSTATUS_VS_DIRTY_UPDATE != "never")
+  RVV_VL_WHEN_AVL_LT_DOUBLE_VLMAX:
+    description: |
+      The value assigned to VL when AVL < 2*VLMAX.
+    schema:
+      type: string
+      enum: ["ceil(AVL/2)", "VLMAX", "custom"]
+  FOLLOW_VTYPE_RESET_RECOMMENDATION:
+    description: |
+      The spec recommends - but does not require - that on reset, vill is set, and remaining bits in vtype and vl are set to 0.
+      If this parameter is set to true, the CSR resets will follow this recommandation. If it is false, the resets will be "UNDEFINED_LEGAL".
```

Typo "recommandation", but perhaps we should use the text from the spec here, which I think is clearer:
```suggestion
      It is recommended that at reset, vtype.vill is set, the remaining bits in vtype are zero, and vl is set to zero.
      If this parameter is set to true, this recommendation is followed. If it is false, at reset the respective fields will be "UNDEFINED_LEGAL".
```